src/erlang_ls.erl

```diff
@@ -64,11 +68,13 @@ lager_config() ->
 
 -spec lager_handlers(string()) -> [any()].
 lager_handlers(LogRoot) ->
+  {ok, LoggingLevel} =
```

Comment:
This will cause a crash. `application:get_env/3` returns the actual value, not a tuple. See:

http://erlang.org/doc/apps/kernel/application.html#get_env-3